driver/main.c

```diff
@@ -1955,6 +1999,10 @@ static int record_event_consumer(struct ppm_consumer_t *consumer,
 			args.syscall_id = event_datap->event_info.syscall_data.id;
 			args.cur_g_syscall_table = event_datap->event_info.syscall_data.cur_g_syscall_table;
 			args.compat = event_datap->compat;
+			if(unlikely(preload_params(&args, event_datap->is_socketcall) == -1))
+			{
+				/* we need to do something, not sure what... */
```

Comment:
we need to handle this case, maybe we could return here (?)

Reply:
Generate a PPME_GENERIC with __NR_socketcall?

Reply:
That is pretty smart indeed!

Reply:
For now, we are returning an error (incrementing the error counter).
Also, this should really never happen i think.